pkg/reconciler/reconciler.go

```diff
@@ -403,6 +414,7 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (res ctrl.Result, err error) {
 		return ctrl.Result{}, nil
 	}
 	if err != nil {
+		log.Error(err, "Failed to lookup resource") //todo: pr
 		return ctrl.Result{}, err
```

Comment:
@joelanford the reconcile shows that are missing logs

Reply:
Controller-runtime logs the error after we return. Right now in SDK, you'll see that a bunch of errors are duplicated because of our log before returning and and c-r's log after returning.

Reply:
We have not logs in the container over what happened. See after all execution of the sample:  (IMO we need the info logs in the operator-container). Also, it is missing the Helm log as well. 

```
 $ kubectl logs deployment.apps/nginx-operator-controller-manager -n nginx-operator-system -c manager
{"level":"info","ts":1592738173.1011891,"logger":"setup","msg":"version information","go":"go1.13.10","GOOS":"linux","GOARCH":"amd64","helm-operator":"0.0.0+git"}
{"level":"info","ts":1592738173.101342,"logger":"setup","msg":"environment variable OPERATOR_NAME has been deprecated, use --leader-election-id instead."}
{"level":"info","ts":1592738173.192791,"logger":"setup","msg":"watching all namespaces"}
{"level":"info","ts":1592738173.500127,"logger":"controller-runtime.metrics","msg":"metrics server is starting to listen","addr":"0.0.0.0:8383"}
{"level":"info","ts":1592738173.5017157,"logger":"controllers.Helm","msg":"Watching resource","group":"helm.sdk.operatorframework.io","version":"v1","kind":"Nginx"}
{"level":"info","ts":1592738173.501763,"logger":"setup","msg":"configured watch","gvk":"helm.sdk.operatorframework.io/v1, Kind=Nginx","chartPath":"./helm-charts/nginx","maxConcurrentReconciles":4,"reconcilePeriod":0}
{"level":"info","ts":1592738173.5017776,"logger":"setup","msg":"starting manager"}
{"level":"info","ts":1592738173.502062,"logger":"controller-runtime.manager","msg":"starting metrics server","path":"/metrics"}
I0621 11:16:13.502010       1 leaderelection.go:242] attempting to acquire leader lease  nginx-operator-system/controller-manager...
I0621 11:16:13.510528       1 leaderelection.go:252] successfully acquired lease nginx-operator-system/controller-manager
{"level":"info","ts":1592738173.5948253,"logger":"controller-runtime.controller","msg":"Starting EventSource","controller":"nginx-controller","source":"kind source: helm.sdk.operatorframework.io/v1, Kind=Nginx"}
{"level":"info","ts":1592738173.6974845,"logger":"controller-runtime.controller","msg":"Starting EventSource","controller":"nginx-controller","source":"kind source: /v1, Kind=Secret"}
{"level":"info","ts":1592738173.7983105,"logger":"controller-runtime.controller","msg":"Starting Controller","controller":"nginx-controller"}
{"level":"info","ts":1592738173.7987888,"logger":"controller-runtime.controller","msg":"Starting workers","controller":"nginx-controller","worker count":4}

```

Reply:
Create an issue for check that: https://github.com/joelanford/helm-operator/issues/14